dashboard/modules/log/test_log.py

```diff
@@ -0,0 +1,109 @@
+import os
+import logging
+import requests
+import socket
+import time
+import traceback
+import html.parser
+import urllib.parse
+
+import ray
+from ray.new_dashboard.tests.conftest import *  # noqa
+from ray.test_utils import (
+    wait_until_server_available, )
+
+os.environ["RAY_USE_NEW_DASHBOARD"] = "1"
+
+logger = logging.getLogger(__name__)
+
+
+class LogUrlParser(html.parser.HTMLParser):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._urls = []
+
+    def handle_starttag(self, tag, attrs):
+        if tag == "a":
+            self._urls.append(dict(attrs)["href"])
+
+    def error(self, message):
+        logger.error(message)
+
+    def get_urls(self):
+        return self._urls
+
+
+def test_log(disable_test_module, ray_start_with_dashboard):
+    @ray.remote
+    def write_log(s):
+        print(s)
+
+    test_log_text = "test_log_text"
+    ray.get(write_log.remote(test_log_text))
+    assert (wait_until_server_available(ray_start_with_dashboard["webui_url"])
+            is True)
+    webui_url = ray_start_with_dashboard["webui_url"]
+    webui_url = webui_url.replace("localhost", "http://127.0.0.1")
+
+    timeout_seconds = 20
```

Comment:
Can you reduce it to 10 seconds? 10 seconds timeout makes test flaky we should break them down instead.

Reply:
OK. 10 seconds is enough.